scripts/delete_db_and_setup.sh

```diff
@@ -21,7 +21,7 @@ if [ $MODE == 'PROD' ]; then
   echo "Please enter your desired email and password."
   $EXEC_ENV python3 manage.py createsuperuser --username admin
 else
-  $EXEC_ENV python3 manage.py createsuperuser --username admin --email admin@examplee.com --no-input
```

Comment:
This was actually intentional in order to preserve the superuser when deleting all fakeusers with the script. But I can live with it both ways since I rarely use the fakeuser-deletion.

Reply:
We can add it again. Without documentation this looks like a typo 😬